### PR TITLE
Improve OTEL docs according to customer feedback

### PR DESCRIPTION
**Pull request by adinauer**
We had a call with an OTEL customer recently. This updates the docs according to their feedback.

Generic:
- Add preface onboarding users with OTEL knowledge to Sentry
- Link `beforeSendTransaction` so users can filter out e.g. health checks

Java specific:
- Also link JAR on MavenCentral

### CODE REVIEW

File: src/platforms/common/performance/instrumentation/opentelemetry.mdx [hunk 1]
```diff
@@ -33,12 +32,94 @@ description: "Using OpenTelemetry with Sentry Performance."
...
 
+</PlatformSection>
+
+<PlatformSection supported={["java"]}>
+
+There are multiple ways of configuring your [OpenTelemetry SDK](https://opentelemetry.io/) to send traces and spans to Sentry.
+
+</PlatformSection>
+
+## Preface
```
Comment (adinauer) [inline src/platforms/common/performance/instrumentation/opentelemetry.mdx:47]: Please helps improve this section to make it accurate and easy to understand.

Comment (HazAT) [reply src/platforms/common/performance/instrumentation/opentelemetry.mdx:47]: image

```
An OpenTelemetry Span can become a Sentry Transaction or Span. The first Span sent through the Sentry SpanProcessor is a Transaction, and any child Span of it is attached to the Transaction by checking the parent Span context. This happens for the OpenTelemetry root Span and any top-most Span in the system. For example, a request from frontend to backend creates an OpenTelemetry root Span and its corresponding Sentry Transaction. The backend request creates a new Sentry Transaction for its OpenTelemetry Span. The Sentry Transaction and Span are linked as a trace for navigation and error tracking.
```

File: src/platforms/common/performance/instrumentation/opentelemetry.mdx [hunk 2]
```diff
@@ -33,12 +32,94 @@ description: "Using OpenTelemetry with Sentry Performance."
...
+
```
Comment (lizokm) [inline src/platforms/common/performance/instrumentation/opentelemetry.mdx:52]:
```suggestion
With Sentry’s OpenTelemetry SDK, an OpenTelemetry `Span` becomes a Sentry `Transaction` or `Span`. The first `Span` sent through the Sentry `SpanProcessor` is a `Transaction`, and any child `Span` gets attached to the first `Transaction` upon checking the parent `Span` context. This is true for the OpenTelemetry root `Span` and any top level `Span` in the system. For example, a request sent from frontend to backend will create an OpenTelemetry root `Span` with a corresponding Sentry `Transaction`. The backend request will create a new Sentry `Transaction` for the OpenTelemetry `Span`. The Sentry `Transaction` and `Span` are linked as a trace for navigation and error tracking purposes.

```

File: src/platform-includes/performance/opentelemetry-install/with-java-agent/without-auto-init/java.mdx [hunk 3]
```diff
@@ -1,4 +1,4 @@
-You can download the latest version of the `sentry-opentelemetry-agent-{{ packages.version('sentry.java.opentelemetry-agent') }}.zip` from [GitHub](https://github.com/getsentry/sentry-java/releases/) which contains the `JAR` file used in this docs page.
+You can download the latest version of the `sentry-opentelemetry-agent-{{ packages.version('sentry.java.opentelemetry-agent') }}.jar` from [MavenCentral](https://search.maven.org/artifact/io.sentry/sentry-opentelemetry-agent). It is also available as `ZIP` containg the `JAR` used on this page on [GitHub](https://github.com/getsentry/sentry-java/releases/).
```
Comment (lizokm) [inline src/platform-includes/performance/opentelemetry-install/with-java-agent/without-auto-init/java.mdx:1]:
```suggestion
You can download the latest version of the `sentry-opentelemetry-agent-{{ packages.version('sentry.java.opentelemetry-agent') }}.jar` from [MavenCentral](https://search.maven.org/artifact/io.sentry/sentry-opentelemetry-agent). It's also available as a `ZIP` containing the `JAR` used on this page on [GitHub](https://github.com/getsentry/sentry-java/releases/).
```

File: src/platforms/common/performance/instrumentation/opentelemetry.mdx [hunk 4]
```diff
@@ -33,12 +32,94 @@ description: "Using OpenTelemetry with Sentry Performance."
...
+<PlatformContent includePath="performance/opentelemetry-setup/with-java-agent/without-auto-init/debugging" />
+
+### Turn Off Exporter Error Messages
+
+<PlatformContent includePath="performance/opentelemetry-setup/with-java-agent/exporter-messages" />
+
+## Using OpenTelemetry Without Any Java Agent
+
+If the Java Agent approach is not for you, you may also manually initialize OpenTelemetry. We have a separate dependency for this use case that allows you to reuse classes used by `sentry-opentelemetry-agent`.
```
Comment (lizokm) [inline src/platforms/common/performance/instrumentation/opentelemetry.mdx:109]:
```suggestion
If the Java Agent approach isn't for you, you can manually initialize OpenTelemetry. We have a separate dependency for this use case that allows you to reuse classes used by `sentry-opentelemetry-agent`.
```

File: src/platforms/common/performance/instrumentation/opentelemetry.mdx [hunk 5]
```diff
@@ -33,12 +32,94 @@ description: "Using OpenTelemetry with Sentry Performance."
...
+
+### Install
+
+<PlatformContent includePath="performance/opentelemetry-install/without-java-agent" />
+
+### Usage
+
+You will have to configure both OpenTelemetry and Sentry to see transactions in Sentry and have errors linked to transactions created by OpenTelemetry.
+
```
Comment (lizokm) [inline src/platforms/common/performance/instrumentation/opentelemetry.mdx:118]:
```suggestion
You'll have to configure both OpenTelemetry and Sentry to see transactions in Sentry and have errors linked to transactions created by OpenTelemetry.

```

File: src/platforms/common/performance/instrumentation/opentelemetry.mdx [hunk 6]
```diff
@@ -33,12 +32,94 @@ description: "Using OpenTelemetry with Sentry Performance."
...
+<PlatformContent includePath="performance/opentelemetry-setup/without-java-agent" />
+
+</PlatformSection>
+
+## Filtering
+
+In case you want to filter out transactions before sending them to Sentry, for example to get rid of health checks, you can use <PlatformLink to="/configuration/options/#before-send-transaction"><PlatformIdentifier name="before-send-transaction" /></PlatformLink> to drop them. <PlatformLink to="/configuration/filtering/#using-platformidentifier-namebefore-send-transaction-">Here you can find an example on how to use it</PlatformLink>.
```
Comment (lizokm) [inline src/platforms/common/performance/instrumentation/opentelemetry.mdx:125]:
```suggestion
If you want to filter out transactions before sending them to Sentry, ( to get rid of health checks, for example), you can use <PlatformLink to="/configuration/options/#before-send-transaction"><PlatformIdentifier name="before-send-transaction" /></PlatformLink> to drop them. <PlatformLink to="/configuration/filtering/#using-platformidentifier-namebefore-send-transaction-">Here's an example</PlatformLink>.
```

**Review by lizokm — APPROVED** [review]
Made some tweaks.

File: src/platforms/common/performance/instrumentation/opentelemetry.mdx [hunk 7]
```diff
@@ -33,12 +32,94 @@ description: "Using OpenTelemetry with Sentry Performance."
...
 
 <PlatformContent includePath="performance/opentelemetry-setup" />
+
+</PlatformSection>
+
+<PlatformSection supported={["java"]}>
+
+## Using `sentry-opentelemetry-agent` With Auto Initialization
+
+If you use `sentry-opentelemetry-agent`, it will look for `SENTRY_DSN` and `SENTRY_PROPERTIES_FILE` environment variables to be defined, and then initialize Sentry automatically. Only your `DSN` and `tracesSampleRate` need to be configured.
```
[the remark this answers is not in the document]
Comment (adinauer) [reply src/platforms/common/performance/instrumentation/opentelemetry.mdx]: Thanks for the review. Kept the first sentence as is as it's not defining the env vars but rather checking whether they're defined.

**Pull request merged**